lectures/12-plonk.tex

```diff
@@ -0,0 +1,234 @@
+\documentclass[../lecture-notes.tex]{subfiles}
+\usepackage[T1]{fontenc}
+\usepackage{lmodern}
+\usepackage{xcolor}
+\usepackage{fix-cm}
+\pgfplotsset{compat=1.18}
+
+\begin{document}
+
+\subsection{Preliminaries}
+
+Before delving into Plonk, let's first consider a few useful proving system gadgets.
+
+Let $\Omega$ be some subset of $\mathbb{F}_p$ of size $k$.
+Let $f \in \mathbb{F}_p^{(\leq d)}[X] \qquad (d \geq k)$ \hfill \begin{tabular}{|c|}\hline Verifier has $f$ \\ \hline \end{tabular} 
+
+Now we need to construct efficient Poly-IOPs for two tasks described in below sections.
+
+\subsubsection{ZeroTest}
+
+Prove that $f$ is identically zero on $\Omega$.
+
+\begin{enumerate}
```

Comment:
It would be cool to draw a flow diagram for that. But that is not necessary as of now